manifests/config.pp

```diff
@@ -87,6 +87,13 @@
     feature   => 'Puppet CA',
     listen_on => $::foreman_proxy::puppetca_listen_on,
   }
+  if $::foreman_proxy::puppetca_modular {
+    foreman_proxy::settings_file { [
```

Comment:
nit: there's no need for a list here and it can just be a string. The code is slightly more readable then.

Reply:
Well, it will be an array eventually when we add the other provider's setting file...